core/block.ts

```diff
@@ -2173,9 +2163,12 @@ export class Block implements IASTNodeLocation, IDeletable {
    * Returns the comment on this block (or null if there is no comment).
    *
    * @returns Block's comment.
+   * @deprecated Use getIcon. To be remove in v11.
    */
   getCommentText(): string | null {
-    return this.commentModel.text;
+    deprecation.warn('getCommentText', 'v10', 'v11', 'getIcon');
```

Comment:
I find it strange we are deprecating `getCommentText` and not `setCommentText`, and would probably prefer to deprecate them at the same time, but awaiting your thoughts

Reply:
Discussed offline, undeprecated `setCommentText`!